duckduckgo/iOS
# Additional pixels for bookmark import errors

Task/Issue URL: https://app.asana.com/0/0/1202428268249559/f
Tech Design URL:
CC:

**Description**:
Adding additional pixels to help narrow down the source of the high import error rate 



**Steps to test this PR**:
1. Import an invalid bookmarks html file (there is one [here](https://github.com/duckduckgo/iOS/blob/develop/DuckDuckGoTests/MockFiles/bookmarks_invalid.html)) 
2. Confirm the error pixels `m_bi_e` and `m_bi_e_p` fire
3. Import a valid bookmarks html
4. Confirm the success pixel `m_bi_s` fires

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
